devenv/local/docker-compose/README.md

```diff
@@ -0,0 +1,38 @@
+# DOCKER COMPOSE INSTRUCTIONS
+
+
+## [1] Build all containers:
+
+###### `sh build.sh`
+
+## [2] Spin up all the containers:
+
+###### `sh up.sh`
+
+
+## [3] Install some testing pkgs:
+
+#### (i) _Postgres Tools_
+
+```
+brew install libpq
+echo 'export PATH="/opt/homebrew/opt/libpq/bin:$PATH"' >> ~/.zshrc
+source ~/.zshrc
+```
```

Comment:
This is very specific install instructions for MacOs with zsh, so if it's necessary it's probably better to just link to a "how to install postgres" page.